### ISSUE TITLE
Breaks accessibility's magnification

### ISSUE THREAD

**Issue by F-i-f** [hxreborn/punch-hole-download-progress#102]
### What's broken?

Other

### Description

When the module is enabled and settings > accessibility > magnification is enabled as well, the two finger pinch/expand magnification zoom doesn't work. Upon disabling the module, the zoom functionality comes back.

### Steps to reproduce

1. In settings, accessibility enable magnification 
2. Enable the module 
3. The two finger pinch for zoom and unzoom doesn't work. 

### Device

Pixel 9 pro XL

### Android version

16

### ROM (if custom)

stock

### LSPosed version

2.0.4 ksu version

**Comment by hxreborn** [comment]
Hi @F-i-f please test latest build from actions and tell me if it's fixed https://github.com/hxreborn/punch-hole-download-progress/actions/runs/27497570411

**Comment by F-i-f** [comment]
Tested the release APK, all seems to work fine, thank you.

**Comment by hxreborn** [comment]
closing as completed, feel free to open again if something doesn't work as expected
